ablator/main/model/wrapper.py

```diff
@@ -542,6 +794,29 @@ def validation_loop(
         subsample: float = 1.0,
         smoke_test: bool = False,
     ) -> dict[str, float]:
+        """
+        Validate the model with the given tag and dataloader.
```

Comment:
Validate the model on data in `dataloader` (which can either be val dataloader - so tag is `val`, or test dataloader - so tag is `test`)